src/lib/VirtualScroller.js

```diff
@@ -290,12 +302,25 @@ class VirtualScroller {
      * @return {void}
      */
     cloneItems(newListEl, oldListEl, start, end) {
-        if (!newListEl || !oldListEl || start < 0 || end < 0) {
+        if (
+            !newListEl ||
+            !oldListEl ||
+            start === null ||
```

Comment:
would `!start` and `!end` suffice here instead of explicitly checking `null` and `undefined`?